src/Entity/Experience.php

```diff
@@ -21,14 +21,14 @@ class Experience
     private ?string $company = null;
 
     #[Assert\Expression(
-        "!this.getEndDate() ?: this.getEndDate() > this.getStartDate()",
+        "( !this.getEndDate() or this.isIsCurrentPosition() === true )?: this.getEndDate() > this.getStartDate()",
```

Comment:
le this.isIsCurrentPosition() ne te renvoie pas déjà un bool ? 
Si oui pas besoin de === true